getting_started/administrators.adoc

```diff
@@ -66,6 +66,12 @@ $ sudo docker run -d --name "origin" \
         openshift/origin start
 ----
 +
+NOTE: If you want to use
+https://github.com/openshift/origin-aggregated-logging/blob/master/README.md[origin
```

Comment:
Any reason this couldn't link to `../install_config/aggregate_logging.html` instead?